src/sysusers/sysusers.c

```diff
@@ -106,6 +114,10 @@ STATIC_DESTRUCTOR_REGISTER(arg_image, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_image_policy, image_policy_freep);
 
 typedef struct Context {
+#if HAVE_AUDIT
+        int audit_fd;
+#endif
```

Comment:
maybe add a struct AuditContext or so for this, and add it to audit-util.h, and simply make it an empty struct in audit-util.h if HAVE_AUDIT is not 1